lib/Rex/Group/Lookup/XML.pm

```diff
@@ -33,7 +33,7 @@ use Rex -base;
 require Exporter;
 use base qw(Exporter);
 use vars qw(@EXPORT);
-XML::LibXML->require;
+use XML::LibXML;
```

Comment:
No, I'm especially fine with dropping support for unsupported distros. I just read that [RHEL4 has extended support till 2017-03-31](https://en.wikipedia.org/wiki/Red_Hat_Enterprise_Linux#Product_life_cycle). Let's find some official info.